### DISCUSSION TITLE
GetBuilder() method for models manager

### DISCUSSION THREAD

**Discussion by webugless** [phalcon/cphalcon#15965] [Ideas]
I think it would be interesting to have more flexibility using builder, like adding new queries to the current builder instead of creating new one eache time. for example, I would like to add new queries after verifiying parameters passed to url, something like
if ($param !== null){
  $this->model->manager->getBuilder()->join(...)
 }

**Comment by niden** [comment]
@TheAlg Can you open an issue for this please?
